### PR TITLE
New package: gnome-usage-3.28.0

### PR DESCRIPTION
**Pull request by Cogitri**

### CODE REVIEW

File: srcpkgs/gnome-usage/template [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Template file for 'gnome-usage'
+pkgname=gnome-usage
+version=3.28.0
+revision=1
+build_style=meson
+hostmakedepends="pkg-config"
+makedepends="gettext-devel glib-devel gtk+3-devel libgtop-devel vala-devel"
```
Comment (maxice8) [inline srcpkgs/gnome-usage/template]: see if libglib-devel can be used instead. If a binary from `glib-devel` needs to be run then add `glib-devel` to hostmakedepends

File: srcpkgs/gnome-usage/template [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+short_desc="A nice way to view information about use of system resources"
```
Comment (maxice8) [inline srcpkgs/gnome-usage/template]: Please avoid superfluous A at the beggining

File: srcpkgs/gnome-usage/template (hunk 1 above)
Comment (Cogitri) [reply srcpkgs/gnome-usage/template]: Oh, yeah, it needs glib-compile-schemas, that's why I didn't use `libglib-devel`

Comment (Cogitri) [reply srcpkgs/gnome-usage/template]: Uhm...I guess this still need libglib-devel as makedepend though?

**Comment by maxice8** [comment]
On Sun, Sep 02, 2018 at 10:42:00AM -0700, Rasmus Thomsen wrote:
> Cogitri commented on this pull request.
> 
> 
> 
> > @@ -0,0 +1,13 @@
> +# Template file for 'gnome-usage'
> +pkgname=gnome-usage
> +version=3.28.0
> +revision=1
> +build_style=meson
> +hostmakedepends="pkg-config"
> +makedepends="gettext-devel glib-devel gtk+3-devel libgtop-devel vala-devel"
> 
> Uhm...I guess this still need libglib-devel as makedepend though?
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/2322#discussion_r214548069

yes if you're compiling against glib you need libglib-devel in makedepends

if you need to use glib stuff like glib-compile-schemas (or whatever it is called) you need it in hostmakedepends.

if you need both you add both

**Pull request merged**